pgjdbc/src/main/java/org/postgresql/jdbc/PgDatabaseMetaData.java

```diff
@@ -2604,12 +2631,28 @@ public int getDatabaseMinorVersion() throws SQLException {
     return connection.getServerMinorVersion();
   }
 
-  public int getJDBCMinorVersion() throws SQLException {
-    return 0; // This class implements JDBC 3.0
+  /**
+   * Retrieves the major JDBC version number for this driver.
+   *
+   * @return JDBC version major number
+   */
+  @Override
```

Comment:
Well, technically speaking, there's non-trivial build step that unzips jdk sources (e.g. https://github.com/pgjdbc/pgjdbc-parent-poms/blob/master/pgjdbc-core-parent/pom.xml#L270-L278) and adds them to javadoc tool, however in it does "just work".